Sources/Classes/RSEventRepository.m

```diff
@@ -187,7 +187,9 @@ - (void) __initiateSDK {
                     if (serverConfig.destinations != nil && serverConfig.destinations.count > 0) {
                         NSArray <RSServerDestination *> *consentedDestinations = self->consentFilterHandler != nil ? [self->consentFilterHandler filterDestinationList:serverConfig.destinations] : serverConfig.destinations;
                         if(consentedDestinations != nil && consentedDestinations.count > 0 ) {
-                            [self->deviceModeManager startDeviceModeProcessor:consentedDestinations andDestinationsWithTransformationsEnabled:[strongSelf->configManager getDestinationsWithTransformationsEnabled]];
+//                            [RSUtils filterConsentedDestinationsInList:[strongSelf->configManager getDestinationsWithTransformationsEnabled] byUsing: consentedDestinations];
+                            [self->deviceModeManager startDeviceModeProcessor:consentedDestinations withConfigManager:strongSelf->configManager];
+//                        andDestinationsWithTransformationsEnabled:[strongSelf->configManager getDestinationsWithTransformationsEnabled] andDestinationsExcludedOnTransformationError: [strongSelf->configManager getExcludedDestinationsOnTransformationsError]];
```

Comment:
Can we remove the comment out code (line 190 & 192).

Reply:
removed it now